src/components/NDABanner/index.tsx

```diff
@@ -48,10 +48,16 @@ const NDABanner = ({ collapsable, landing, className }: NDABannerType) => {
                       iconPosition="left"
                       toggleClassName="margin-bottom-2"
                     >
-                      <p className="padding-0 margin-0">{t('body')}</p>
+                      <p>
+                        <span className="text-bold">{t('caveat')}</span>{' '}
+                        <span>{t('body')}</span>
+                      </p>
```

Comment:
```suggestion
                      <p className="margin-y-0">
                        <span className="text-bold">{t('caveat')}</span>{' '}
                        <span>{t('body')}</span>
                      </p>
```

Reply:
The spacing for the NDA banner is a bit off from Figma. Removing the margin from the paragraph tag makes it better